api/src/controllers/website.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * This module contains handlers for the "clientWebsite" route.
+ * @module website
+ */
+
+const Website = require('../models/website.model');
+
+const { fillErrorObject } = require('../middleware/error');
+
+/**
+ * Handle GET request from /clientWebsite/:team_id
+ * @param {*} req request object, containing teamId
+ * @param {*} res response object
+ * @returns 200: the team's website info was found
+ * @returns 404: team's website info is not found
+ * @returns 500: server error
+ */
+function getWebPageDetails(req, res) {
+  Website.findOne({
+    teamId: req.params.team_id,
+  })
+    .then((websiteInfo) => {
+      if (!websiteInfo) {
+        return res
+          .status(404)
+          .send(
+            'No Website info found for this Team (This is likely because they have not added any website information yet)'
+          );
+      }
+      return res.status(200).send(websiteInfo);
+    })
+    .catch((error) => {
+      return res.status(500).json(`Server Error: ${error.message}`);
```

Comment:
Can this be turned to async-await to make the code cleaner and avoid the nesting?

Reply:
We can do the refracting in a separate ticket. Will merge this to main for now.

Reply:
Was the commnent by @vgoh1 resolved?
https://github.com/Researchify/Researchify/pull/135#issuecomment-890447160
@kdav108